app/controllers/api/v1/authentication_controller.rb

```diff
@@ -183,7 +200,7 @@ def render_error(error_message)
       end
 
       def authenticated_user
-        warden.authenticated? && @user.allowed_login?(app_name)
+        warden.authenticated? && (is_browse_app? || @user.allowed_login?(app_name))
```

Comment:
Can we remove dependency on `is_browse_app?` - is there a better way to do this?

Reply:
I think we should adjust the permissions so that users are allowed to login on the browse app... rather than adding `is_browse_app` here...

Reply:
See ```User#allowed_login?(app_name)``` and add BROWSE_APP to the list of apps that ```return true```